en/publish/setup-native-development.md

```diff
@@ -44,13 +44,46 @@ After installing Android Studio, refer to the official documentation and open th
 
 ![Sdk manager](setup-native-development/sdk-manager.jpg)
 
-> **Note**: currently, the version of Android Gradle supported by Creator is **4.10.x**, and the corresponding Gradle Tool version is **3.x** (v3.2.0 is recommended). When using Android Studio to compile, please **do not** upgrade when prompted to upgrade the Android Gradle Plugin.
+### Android Gradle Upgrade
 
-![](setup-native-development/gradle-update.png)
+Since we upgraded the Android Gradle version in Creator v2.4.7, it is necessary to explain it in the following two situations.
+
+#### Before Creator v2.4.7
+
+The supported version of Android Gradle before Creator v2.4.7 is **4.10.x**, and the corresponding Gradle Tool version is **3.x** (v3.2.0 is recommended). When using Android Studio to compile, please **do not** upgrade when prompted to upgrade the Android Gradle Plugin.
+
+![gradle-update](setup-native-development/gradle-update.png)
 
 - View the file path for the Android Gradle version as: `YourProject\build\jsb-link\frameworks\runtime-src\proj.android-studio\build.gradle`
 - View the file path for the Gradle Tool version as: `YourProject\build\jsb-link\frameworks\runtime-src\proj.android-studio\gradle\wrapper\gradle-wrapper.properties`
 
+#### Creator v2.4.7 and later
+
+Creator v2.4.7 upgrades the Android Gradle version to **6.7.1** and the corresponding Gradle Tool version to **4.2.2**. This upgrade mainly requires attention to the **File Directory** and **Old Functionality Module** sections.
+
+- **File Directory**
+
+  All Android projects with `instant-app` may encounter problems due to the following adjustments to the file directory of `instantApp` for this upgrade:
+
+    - Removed `game` and merged the project settings from `build.gradle` into `instantapp`.
+
+    - Upgraded `gradle plugin`, the original lower version of the plugin will have renaming problem in the higher version, for example, the developer may need to add `useAndroidX` to enable the original old plugin.
+
+      > **Note**:
+      >
+      > 1. On the **File Directory**, deprecate the `instantapp` and `game` modules and merge them into a new `instantapp` module. New features that were to be added to the deprecated module will need to be added to the new `instantapp` module after the upgrade.
+      > 2. Since the original `instantapp` module's instant app functionality has been merged into the base application module, to compile the instant app, set the project as `instantapp`, i.e. **manually** check the option in the following image:
```

Comment:
```suggestion
      > 2. Since the original `instantapp` module's instant app functionality has been merged into the base app module, to publish to the Android Instant platform, need to set the project as `instantapp` in Android Studio: click **Run -> Edit Configurations** in the top menu bar, then check **Deploy as instant app** in the panel that opens, as shown in the following image:
```